mongodb/build.gradle

```diff
@@ -0,0 +1,11 @@
+version = '0.1-SNAPSHOT'
+
+dependencies {
+    compile 'org.mongodb:mongodb-driver-sync:3.10.1'
+    testCompile "org.testcontainers:testcontainers:1.11.2"
+    testCompile group: 'javax.cache', name: 'cache-api', version: '1.1.0'
```

Comment:
We already have this on the root project as a dependeny. See https://github.com/hazelcast/hazelcast-jet-contrib/blob/master/build.gradle#L51